### DISCUSSION TITLE
destinations/oversea/osaka_1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#574] [Announcements]
# destinations/oversea/osaka_1

“到了大阪才发现时差有一个小时” 从关西机场出来时，空气是凛冽的。这就是日本给我的第一印象。有点喜欢这种感觉，凛冽的空气让大脑很清醒。 solar 没有做任何关于关西机场到大阪市区的交通攻略，这条路线在Google Map上只是一条简单的线条。机场内的指示牌，基本全是汉字。跟着汉字，找到了前往大阪市区的南海电铁。 “这不就是记忆里《铁胆火车侠》的南海忍...

https://skywalkervenne.cloud/destinations/oversea/osaka_1.html

**Comment by skywalkervenne** [comment]
test

**Reply by Mister-Hope** [reply]
# See Repo README
